INSTALL.rst

```diff
@@ -1,6 +1,249 @@
 Installing scikit-image
 =======================
 
+These instructions should help you get started using ``scikit-image``.
+
+We outline 4 different ways which we recommend using to get started with image
+processing and scikit-image.
+
+1. Scientfic Python Distributions -- Recommended
+2. ``pip`` based installations
+3. ``conda`` based installations
+4. System-wide package manager
+
+After your installation, you should be able to run the following code
+from a python console or jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+Should you have any questions getting started using ``scikit-image`` after
+reading this guide, please open an issue on
+`github <https://github.com/scikit-image/scikit-image/issues>`_
+to get additional help.
+
+For instructions on how to install scikit-image from source, please see the
+contribution guide.
+.. TODO: cross reference the contribution guide.
+
+1. Scientfic Python Distributions
+---------------------------------
+
+``scikit-image`` comes pre-installed with several Python distributions
+including:
+
+- `Anaconda <https://www.anaconda.com/distribution/>`_
+- `Python(x,y) <https://python-xy.github.io/>`_
+- `WinPython <https://winpython.github.io/>`_
+
+These have the advantage of comming pre-installed with many other useful
+packages for scientific computing and image analysis. Follow the instructions
+on the respective websites to get going.
+
+Note that the version of ``scikit-image`` installed with these distributions
+may different than the one you expect. This should not stop you from getting
+started with your project. We simply suggest you check the installed version of
+``scikit-image`` with the code snippet included at the top of this guide.
+
+
+2. ``pip`` based installation
+-----------------------------
+
+To install ``scikit-image`` with `pip <https://pip.pypa.io/en/stable/>`_, you
+may use the following commands
+
+.. code-block:: bash
+
+  # Update pip to a more recent version
+  python3 -m pip install -U pip
+  # Install scikit-image
+  python3 -m pip install -U scikit-image
+
+
+This is the easiest way of obtaining the latest released version of
+``scikit-image``.
+
+We also offer an easy way to install related packages. These are often included
+in scientific python distributions. To install many of them using ``pip`` use
+the command:
+
+.. code-block:: bash
+  python3 -m pip install scikit-image[optional]
+
+
+A note on virtual environments
```

Comment:
I think that this must be the introduction of this section

Reply:
After teaching and working closely with about 10 new developers in python, i've learned that:

1. Until they have 2 projects going on at the same time, they just don't care about virtual envs.
2. If their work / course tells them to get scikit-image going, they are happy to "get it going once" then later, when their system breaks, they will work on learning about virtual environments.